src/main/java/org/broadinstitute/hellbender/utils/MathObjectAsserts.java

```diff
@@ -16,6 +18,28 @@
 
     private MathObjectAsserts() {}
 
+    public static void assertNDArrayEquals(final INDArray actual, final INDArray expected,
```

Comment:
because I'd like to have control over both relative and absolute errors. The INDArray implementation only deals with absolute errors.